policyengine_us/variables/gov/hud/income_level_factor/hud_especially_low_income_factor.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class hud_especially_low_income_factor(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "HUD Especially Low income factor"
+    unit = USD
+    documentation = "Especially Low income factor for HUD programs"
+    definition_period = YEAR
+
+    def formula(spm_unit, period, parameters):
+        p_hud = parameters(period).gov.hud.ami_limit
+        size = spm_unit("spm_unit_size", period)
+        size_limit = p_hud.family_size
+        size_limit_excess = p_hud.per_person_exceeding_4
+        size_exceeding_4 = max_(size - 4, 0)
+        size_capped_at_4 = min_(size, 4)
+        return (
+            size_limit.ESPECIALLY_LOW[size_capped_at_4]
```

Comment:
same for the excess and other files - in general please keep the `p.` in where reasonable to indicate it's a parameter
```suggestion
            p.family_size.ESPECIALLY_LOW[size_capped_at_4]
```